.sphinx/conf.py

```diff
@@ -0,0 +1,36 @@
+# Project config.
+project = "LXD"
+author = "LXD contributors"
+
+with open("../shared/version/flex.go") as fd:
+    version = fd.read().split("\n")[-2].split()[-1].strip("\"")
+
+# Extensions.
+extensions = [
+    "myst_parser",
+    "sphinx_tabs.tabs",
+    "sphinx_reredirects"
+]
+
+myst_enable_extensions = [
+    "substitution",
+    "deflist",
+    "linkify"
+]
+
+# Setup theme.
+html_theme_path = ["themes"]
+html_theme = "vanilla"
+html_show_copyright = False
```

Comment:
Why hide the copyright? This makes the "Show source" look very weird.
Also, we should set [html_last_updated_fmt](https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-html_last_updated_fmt) to display the update time/date, which makes it easy to spot if something is going wrong in publishing (for us) or whether a version is still being maintained (for users).